### PR TITLE
Add initial runbook entries for AM's alerts

### PR DESCRIPTION
**Pull request by douglascamata**
Some additional reformatting happened because the TOC generation (which is automatic) now changed the way it's writing markdown lists.

This is partially based on https://runbooks.prometheus-operator.dev/runbooks/alertmanager.

### CODE REVIEW

File: docs/sop/observatorium.md [hunk 1]
```diff
@@ -1802,6 +1810,177 @@ The configuration of the instances of Thanos Receive are not same with Receive C
 
 ---
 
+# Observatorium Alertmanager Alerts
+
+## AlertmanagerFailedReload
+
+### Impact
+
+For users this means that their most recent update to alerts might not be currently in use. Ultimately, this means some of the alerts they have configured may not be firing as expected. Subsequente updates to Alertmanager configuration won't be picked up until the reload succeeds.
```
Comment (philipgough) [inline docs/sop/observatorium.md:1819]: Subsequente -> subsequent (language typo? :) )

File: docs/sop/observatorium.md [hunk 2]
```diff
@@ -1802,6 +1810,177 @@ The configuration of the instances of Thanos Receive are not same with Receive C
...
+
+## AlertmanagerFailedToSendAlerts
+
+### Impact
+
+For users, no impact since another instance of Alertmanager in the cluster should be able to send the notification, unless `AlertmanagerClusterFailedToSendAlerts` is also triggered.
+
+### Summary
+
+One of the Alertmanager instances in the cluster cannot send alerts to integrations.
```
Comment (philipgough) [inline docs/sop/observatorium.md:1873]: integrations -> receivers

File: docs/sop/observatorium.md [hunk 3]
```diff
@@ -1802,6 +1810,177 @@ The configuration of the instances of Thanos Receive are not same with Receive C
...
+
+`high`
+
+### Access Required
+
+- Console access to the cluster that runs Observatorium
+
+### Steps
+
+- In the OSD console for the affected cluster, find the Alertmanager Route. Check that it correctly points to the Alertmanager Service. Check that the Service correctly points to the **all** the Alertmanager pods. Open the Route's address, go to the "Status" tab, and note the IP addresses of the discovered Alertmanager instances. Check if they match the addresses of **all** the Alertmanager pods, none should be missing or mismatching.
```
[the remark this answers is not in the document]
Comment (douglascamata) [reply docs/sop/observatorium.md:1863]: The mention to the route there is only to find and open the Alertmanager UI, nothing else.

[the remark this answers is not in the document]
Comment (philipgough) [reply docs/sop/observatorium.md:1863]: ok, I think that makes more sense. There is probably some kubectl command we could use to achieve same but this is fine too in that case.

**Review by philipgough — APPROVED** [review]

**Review by saswatamcode — APPROVED** [review]
LGTM mod @philipgough's comments.

Really wish we could import this somehow, similar to mixin.

File: docs/sop/observatorium.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (douglascamata) [reply docs/sop/observatorium.md:1863]: FYI I'm making the text clearer that the purpose of finding the route is to open the AM UI.

**Pull request merged**